outputs/email.go

```diff
@@ -13,10 +13,40 @@ import (
 	"github.com/aquasecurity/postee/v2/layout"
 )
 
+type EmailSender interface {
```

Comment:
name is same as `emailSender` defined below. It doesn't improve readability of code.

Reply:
fixed, it's been simplified as per the suggestion.